cmd/root_cmd.go

```diff
@@ -59,6 +60,7 @@ func Prepare() *cobra.Command {
 	snapshotCmd.Flags().String("target-url", "", "Target URL")
 	snapshotCmd.Flags().StringSlice("tables", nil, "List of tables to snapshot, in the format <schema>.<table>. If not specified, the schema `public` will be assumed. Wildcards are supported")
 	snapshotCmd.Flags().Bool("reset", false, "Wether to reset the target before snapshotting (only for postgres target)")
+	snapshotCmd.Flags().Bool("profile", false, "Wether to produce CPU and memory profile files, as well as exposing a /debug/pprof/ endpoint on localhost:6060")
```

Comment:
it looks like a typo at the first glance but it's "wether" for other flags as well 🤔

Reply:
Ah I'll fix it 👍